docs/contributing.md

```diff
@@ -0,0 +1,13 @@
+# Contributing
```

Comment:
A `CONTRIBUTING.md` (all caps) file is typically expected to exist at the root of a repo.